### PR TITLE
Gh1059

### PR DESCRIPTION
**Pull request by davidcarlisle**
# Internal housekeeping

## Status of pull request

update  to base and amsmath,  adding `\ignorespaces` to `\eqno`

@PhelypeOleinik  you'd suggested also `\@ignoretrue` but I don't think
that's needed, or at least I couldn't think of a plausble example where
`\if@ignore` is checked here.

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/doc/ltnews37.tex [hunk 1]
```diff
@@ -584,7 +584,13 @@ \subsection{Correct handling of hooks with only `next' code}
 %
 \githubissue{1052}
 
-
+\subsection{Ignoring space after \texttt{\$\$}}
+Space is normally ignored after a closing \texttt{\$\$},
+but internal latex font handling code could interfere if
```
Comment (FrankMittelbach) [inline base/doc/ltnews37.tex:589]: \LaTeX{}

File: required/amsmath/amsmath.dtx [hunk 2]
```diff
@@ -6261,9 +6261,10 @@ Cannot use `split' here;\MessageBreak trying to recover with `aligned'}%
 %    Standard \latex/ provides three environments for one-line equations:
 %    \cn{[}\cn{]}, \env{equation}, and \env{displaymath}. We add
 %    \env{equation*} as a synonym for \env{displaymath}.
+% \changes{v2.17o}{2023/05/13}{Do not error if \cs{eqno} not primitive}
```
Comment (FrankMittelbach) [inline required/amsmath/amsmath.dtx:6264]: reference  the issue in the changes entry

**Review by FrankMittelbach — APPROVED** [review]
two minor comments on documentation

File: required/amsmath/amsmath.dtx
Comment (FrankMittelbach) [inline required/amsmath/amsmath.dtx:1]: isn't there a changes.txt in that directory too? if so it should probably list the change

**Review by PhelypeOleinik — APPROVED** [review]

**Pull request merged**